app/internal/user/availability.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
```

Comment:
Why is this file nested in a user directory? This should just be user.py

Reply:
yam asked for this :)